frontend/src/screens/SignUpScreen.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { Form, Button, Row, Col } from 'react-bootstrap'
+import SignInContainer from '../components/SignInContainer'
+
+
```

Comment:
For the signup screen, we also need an additional field to ask if the user intends to create a student or tutor account. If possible, it should be a radio group with only student or tutor option for the account.